app/Http/Livewire/Components/Header.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Livewire\Components;
+
+use App\Http\Livewire\Traits\InteractWithModal;
+use App\Models\Configs;
+use Illuminate\Support\Facades\Config;
+use Livewire\Component;
+
+class Header extends Component
+{
+	use InteractWithModal;
+
+	/**
+	 * @var string
+	 */
+	public $title = '';
```

Comment:
```suggestion
	public string $title = '';
```
Please add a PHPdoc comment what this attribute is. Title of what?